.travis.yml

```diff
@@ -1,10 +1,29 @@
 ---
 language: python
-python:
-  - '3.7'
+jobs:
+  include:
+    - name: "Python 3.7 on Xenial Linux"
+      python: 3.7           # this works for Linux but is ignored on macOS or Windows
```

Comment:
Kudos for putting down a comment why it needs to be made like that!

I wonder it it wouldn't be enough to put one block explaining that above the jobs so it isn't repeated below? but that's entirely up to you, it's all a matter of personal preference I suppose.

Reply:
I don't have a strong preference either way. I deduplicated the comment. Are you happy with it?

Reply:
LGTM!